### PR TITLE
Add text comment & test for reset_clusters function

### PR DESCRIPTION
**Pull request by stevenhua0320**

### CODE REVIEW

File: src/diffpy/srmise/tests/test_dataclusters.py [hunk 1]
```diff
@@ -32,3 +33,35 @@ def test___eq__():
             print(f"not-equal test failed on {attr_key}")
             assert not expected == actual
         attributes.update({attr_key: reset})
+
+
+# For reset clusters test, we have two test cases:
+# Precondition: DataClusters object should be a valid object.
+# Case (1): x and y are non-empty with positive res, reset_clusters would reset clusters to largest y arg
```
Comment (sbillinge) [inline src/diffpy/srmise/tests/test_dataclusters.py:40]: it is not clear to me what this reset method is doing.  These tests don't seem to be testing anything.

**Review by sbillinge — COMMENTED** [review]
pls see comment.

File: src/diffpy/srmise/tests/test_dataclusters.py [hunk 2]
```diff
@@ -35,33 +35,25 @@ def test___eq__():
...
-                "input_res": 0,
-            },
-            DataClusters(np.array([]), np.array([]), 0),
-        ),
     ],
 )
 def test_reset_clusters(inputs, expected):
-    actual = DataClusters(x=inputs["input_x"], y=inputs["input_y"], res=inputs["input_res"])
+    actual = DataClusters(x=inputs["x"], y=inputs["y"], res=inputs["res"])
+    expected.reset_clusters()
```
Comment (sbillinge) [inline src/diffpy/srmise/tests/test_dataclusters.py:56]: it looks as if you are not testing anything again.  Remember, expected should be what you expect the function to do.  so if the function is `sum(x,y)` then `actual = sum (2, 3)` but `expected = 5`, not `expected = sum(2,3)`

**Review by sbillinge — COMMENTED** [review]
please see inline.

File: src/diffpy/srmise/tests/test_dataclusters.py (hunk 2 above)
Comment (stevenhua0320) [reply src/diffpy/srmise/tests/test_dataclusters.py:56]: I think reset_clusters we should use the similar approach to the test constructors. This function's behavior is so close to the setdata function, it just did not reset the x, y and res attributes.

**Comment by sbillinge** [comment]
@stevenhua0320  there is currently nothing useful here, so I am just closing this.  You can write some more tests on anyything you like as practice if you like, but I think it is better for you to use the code to exdtract distance lists from some datasets so you can see how the code works and what it does.   Then we can decide what to do about bringing it up to date.b

**Pull request closed without merging**